alignment_report.Rmd

```diff
@@ -51,10 +49,10 @@ total_sequenced_libraries <- nrow(unique(sample_sheet[, c("strain", "id")]))
 
 # Load coverage data
 mapped_cov <- data.table::fread(fs::path_join(c(STRAIN_PATH, "mqc_mosdepth-coverage-per-contig_1.txt"))) 
-   
+
 mapped_cov$coverage <- rowMeans(mapped_cov[,c("I", "II", "III", "IV", "V")], na.rm=TRUE)
```

Comment:
Are we missing `X`?

Reply:
I think I forgot to add it.

Reply:
Also chromM?

Reply:
I forgot about X... Added that back. `7285321`

I looked it up, mitochondria coverage is generally an order of magnitude higher than chromosomes (since there are many mitochondria in each cell). We mostly want to know the coverage on main chromosomes, right? So left it out.